routes/v1/chatRoutes.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router();
+const { chatController } = require('../../controllers');
+const { authMiddleware } = require('../../middlewares');
+
+router.post(
+  '/create_chat',
```

Comment:
There is no need to add create_chat path in post request jut POST method with /chats is enough